src/app/container/view/view.component.ts

```diff
@@ -51,18 +55,30 @@ export class ViewContainerComponent extends View implements OnInit {
     private viewService: ViewService,
     private sessionQuery: SessionQuery,
     private containerService: ContainerService,
+    private containersService: ContainersService,
     private containertagsService: ContainertagsService,
     private hostedService: HostedService,
     private toolQuery: ToolQuery,
-    private matDialog: MatDialog
+    private matDialog: MatDialog,
+    private alertService: AlertService
   ) {
     super(dateService, alertQuery);
   }
 
   setMode(mode: TagEditorMode) {
     this.versionModalService.setVersion(this.version);
-    this.versionModalService.setCurrentMode(mode);
-    this.matDialog.open(VersionModalComponent, { width: '600px' });
+    forkJoin([
+      this.containersService.getTestParameterFiles(this.tool.id, this.version.name, ToolDescriptor.TypeEnum.CWL),
+      this.containersService.getTestParameterFiles(this.tool.id, this.version.name, ToolDescriptor.TypeEnum.WDL),
+    ]).subscribe(
+      (items) => {
+        this.versionModalService.setCurrentMode(mode);
+        this.matDialog.open(VersionModalComponent, { width: bootstrap4mediumModalSize });
```

Comment:
Was going to complain about introducing new Bootstrap dependencies, since our goal is to get rid of it, but this is just our constant. :)